src/test/java/org/junit/tests/assertion/MultipleFailureExceptionTest.java

```diff
@@ -67,6 +68,16 @@ public void assertEmptyThrowsMutipleFailureExceptionForManyThrowables() throws E
         }
     }
 
+    @Test
+    public void assertEmptyErrorListConstructorFailure() throws Exception {
+        try {
+            new MultipleFailureException(Collections.<Throwable>emptyList());
+            fail();
```

Comment:
The `fail()`methid throws `AssertionError` which is an `Exception`. Either use `ExpectedException` or change your catch clause to catch the exact exception you are expecting